docs/reference/commandline/service_create.md

```diff
@@ -42,6 +42,7 @@ Options:
       --help                               Print usage
       --host list                          Set one or more custom host-to-IP mappings (host:ip)
       --hostname string                    Container hostname
+      --isolation isolation                defines the isolation mode (hyperv, process)
```

Comment:
Here and below, this help text could just b e "Isolation mde for the service". For most commands we don't list out potential values here. @thaJeztah WDYT?